dns/bind/src/opnsense/service/conf/actions.d/actions_bind.conf

```diff
@@ -27,3 +27,10 @@ command:/usr/local/opnsense/scripts/OPNsense/Bind/dnsbl.sh
 parameters: %s
 type:script
 message:fetching DNSBLs
+
+[dnsblcron]
+command:/usr/local/opnsense/scripts/OPNsense/Bind/dnsbl.sh;/usr/local/etc/rc.d/named restart 
+parameters: %s
```

Comment:
does this really have a parameter?

Reply:
yes, and should reload suffice anyway? bad bouncing cache...

Reply:
yep, stupid c+p :)